pubspec.yaml

```diff
@@ -24,53 +11,34 @@ dependencies:
   flutter:
     sdk: flutter
 
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.0
 
+  dependencies:
+  flutter_svg: ^0.14.0
```

Comment:
Вы просто решили использовать версию пониже, или просто нашли пример? Для использования актуальных версий можете зайти сразу на https://pub.dev/packages/flutter_svg/install, тут обычно уже минимальная информация, необходимая для использования и подключения

Reply:
Версия ^0.19.0 Завелась При попытке поставить актуальную 0.19.2+1 была ругань:

The current Flutter SDK version is 1.22.5.

Because places depends on flutter_svg >=0.19.2 which requires Flutter SDK version >=1.24.0-6.0.pre, version solving failed.
pub get failed (1; Because places depends on flutter_svg >=0.19.2 which requires Flutter SDK version >=1.24.0-6.0.pre, version solving failed.)